src/components/CommentCards.jsx

```diff
@@ -37,15 +44,42 @@ export const CommentCards = ({ reviewId }) => {
         let realDate = `${date[2]}/${date[1]}/${date[0]}`;
         return (
           <li key={comment.comment_id}>
-            
             <div id="comment-sub-info">
               <h4 className=" sub-info emphasise">@{comment.author}</h4>
               <p className=" sub-info">{realDate}</p>
             </div>
             <p className="comment">"{comment.body}"</p>
 
             <CommentVotes votes={comment.votes} />
-            <Divider id='divider'/>
+            {isLoggedIn && user.username === comment.author ? (
```

Comment:
If you desire to be logged in to see any comments at all - some indication to the user would be great.
Can we only see comments by the logged in user here?

Reply:
Would also be nice to indicate that we can only post as "jessjelly" at the moment - now that you have your user instate, you might as well pass that to the post function